src/app/legacy/containers/ConsentBanner/Banner/cookie.amp.jsx

```diff
@@ -162,6 +151,39 @@ const OptionsItem = styled.li`
     border: ${BORDER_WIDTH} solid ${props => props.theme.palette.CONSENT_ACTION};
     color: ${props => props.theme.palette.CONSENT_ACTION};
   }
+
+  &.hide {
+    width: 44px;
+    height: 44px;
+    position: absolute;
+    top: 0;
+    right: 0;
```

Comment:
Just thinking, this may need to use the `direction` aware CSS property [inset-inline-start](https://developer.mozilla.org/en-US/docs/Web/CSS/inset-inline-start). This means it will be `right:0` on left-to-right services, and `left:0` on right-to-left services.

Worth double checking with UX if it is the intention for the 'X' close button to be on the left for RTL services though!

Reply:
Good thought, checked with UX and they reckon its okay to be on the right side for both for now, but they will follow up with editorial later to see what they think so could be future work there (maybe could bundle it into a pr when we add translations)